src/class-block-converter.php

```diff
@@ -200,7 +201,11 @@ protected function img( DOMElement|DOMNode $element ): ?Block {
 			$image_src = $element->getAttribute( 'src' );
 		}
 
-		$image_src = $this->upload_image( $image_src, $alt ?? '' );
+		try {
+			$image_src = $this->upload_image( $image_src, $alt ?? '' );
```

Comment:
With the changes in this PR, the null-coalescing on `$image_url` below does nothing. It's also still possible to create a block with an empty `src`.

Reply:
Good catch. The null coalescing on alt is also unneeded, as `getAttribute` will always return a string. I've fixed this up.